website/emails/newsletter/2026-05-newsletter.html

```
@@ -0,0 +1,1870 @@
+										font-weight: 400;
+										font-size: 22px;
+										line-height: 30px;
+										color: #191a19;
+									"
+								>
+									Milestones turn good intentions into deadlines. <span style="color: #01579b">We actually like them.</span> 500 recipients by June 2026, set by our board. 1,000 by April 2027, set with our platform partner Somaha. Both within reach, I would say. One essential part of it is <span style="color: #01579b">finding the right NGO local partners</span>. Here's where we stand:
+
+								</p>
+							</td>
+						</tr>
+						<tr>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </tr> ], start tag match failed [ <tr> ] on line 333.

(tag-pair)